scripts/content/script-library.php

```diff
@@ -159,26 +159,39 @@ function normalize_crisis_number($input, $plain = FALSE): string {
  *   The node to serialize.
  * @param string $message
  *   The log message for the new revision.
+ * @param bool $new
+ *   Whether the revision should be created or updated.
  *
  * @return int
  *   Either SAVED_NEW or SAVED_UPDATED, depending on the operation performed.
  */
-function save_node_revision(NodeInterface $node, $message): int {
+function save_node_revision(NodeInterface $node, $message = '', $new = FALSE): int {
```

Comment:
I don;t think we have any scripts waiting in the wings to run, but I think this is risky, since it changes the default behavior of this function.  It used to create a new revision by default, not it does not create one by default.

To maintain backward compatibility don't we want the default to be true.
```suggestion
function save_node_revision(NodeInterface $node, $message = '', $new = TRUE): int {
```

Reply:
This has been changed.